From 8a3a389d46f3eb809d5e2d5e9d843eb24da94c05 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Wed, 16 Oct 2024 00:22:29 +0100
Subject: [PATCH] Remove locations without a METAR station

The following locations have been dropped from the METAR list:

- Los Angeles / USC Campus Downtown
- Brunswick, Naval Air Station
---
 data/Locations.xml | 16 +---------------
 1 file changed, 1 insertion(+), 15 deletions(-)

diff --git a/data/Locations.xml b/data/Locations.xml
index f4d1e6c6..79f22ac6 100644
--- a/data/Locations.xml
+++ b/data/Locations.xml
@@ -23222,7 +23222,7 @@
           <!-- A city in Ukraine: https://en.wikipedia.org/wiki/Kyrylivka -->
           <_name>Kyrylivka</_name>
           <coordinates>46.370000 35.366667</coordinates>
-        </city>    
+        </city>
       </state>
       <state>
         <!-- Ukraine Zhytomyr Oblast (region/province) -->
@@ -33549,13 +33549,6 @@
           <radar>lax</radar>
           <coordinates>33.811667 -118.146389</coordinates>
         </location>
-        <location>
-          <name>Los Angeles / USC Campus Downtown</name>
-          <code>KCQT</code>
-          <zone>CAZ041</zone>
-          <radar>lax</radar>
-          <coordinates>34.016667 -118.283333</coordinates>
-        </location>
         <location>
           <name>Whiteman Airport</name>
           <code>KWHP</code>
@@ -39057,13 +39050,6 @@
           <radar>btv</radar>
           <coordinates>44.450000 -68.366667</coordinates>
         </location>
-        <location>
-          <name>Brunswick, Naval Air Station</name>
-          <code>KNHZ</code>
-          <zone>MEZ024</zone>
-          <radar>btv</radar>
-          <coordinates>43.900278 -69.935000</coordinates>
-        </location>
         <location>
           <name>Caribou Municipal Airport</name>
           <code>KCAR</code>
-- 
GitLab

